packages/gui/src/components/nfts/NFTProperties.tsx

```diff
@@ -87,7 +87,10 @@ export default function NFTProperties(props: NFTPropertiesProps) {
   const { attributes } = props;
 
   const valueAttributes = useMemo(() => {
-    return attributes?.filter((attribute) => !isRankingAttribute(attribute));
+    if (Array.isArray(attributes)) {
```

Comment:
you can use when attributes === undefined

return attributes?.filter((attribute) => !isRankingAttribute(attribute)) ?? [];

or when attributes is null

return attributes?.filter((attribute) => !isRankingAttribute(attribute)) || [];

instead

Reply:
So the issue wasn't that attributes was undefined, but rather, it was an object instead of an array. i.e.
`{trait_type: 'un1qu3EXE', value: 'm32k13 7233'}`

instead of:

`[{trait_type: 'un1qu3EXE', value: 'm32k13 7233'}]`

The failure occurred because `filter()` was being called on an object

Reply:
I see.